modules/microshift-nw-router-config-ports.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-router-ports_{context}"]
+= Configuring router ports
+
+You can control which ports your devices use by configuring the router ingress fields.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+//Q: Any other prereqs?
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` port values in the `ingress.ports.http` and `ingress.ports.https` fields to the ports you want to use:
++
+.Example `config.yaml` router settings
+[source,yaml]
+----
+apiServer:
+  advertiseAddress: 10.44.0.0/32
+#....
+ingress:
+  listenAddress: <1>
+    - "" <2>
+  ports:
+    http: 80 # <3>
+    https: 443 # <4>
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed
+  status: Managed # <5>
+# ....
+----
+<1> Defaults to the entire network of the host.
+<2> Valid values are a single IP address or a list of IP addresses.
```

Comment:
valid values for the listenAddress are single IP address(or the interface name) or list of IP addresses(and interface names)

example for the listenAddress:
```
ingress:
    listenAddress:
        - "10.1.235.168"
        - "ens4"
```